packages/database/src/migrations/20210415062117-DataSourceCodeCleanUpLaosEoc-modifies-data.js

```diff
@@ -0,0 +1,176 @@
+'use strict';
+
+import { arrayToDbString } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const dengueNoWarningCase = { Dengue_Fever_Without_Warning_Signs_Case: 'Dengue_no_warning_case' };
+const dengueWithWarningCase = { Dengue_Fever_With_Warning_Signs_Case: 'Dengue_with_warning_case' };
+const dengueSevereCase = { Severe_Dengue_Case: 'Dengue_severe_case' };
+const dengueNoWarningDeath = {
+  Dengue_Fever_Without_Warning_Signs_Death: 'Dengue_no_warning_death',
+};
+const dengueWithWarningDeath = {
+  Dengue_Fever_With_Warning_Signs_Death: 'Dengue_with_warning_death',
+};
+const dengueSevereDeath = { Severe_Dengue_Death: 'Dengue_severe_death' };
+const malariaCase = { Total_Positive_Malaria_Cases: 'Malaria_case' };
+const malariaDeath = { Total_Malaria_Deaths: 'Malaria_death' };
+const measlesCase = { Total_Positive_Measles_Cases: 'Measles_case' };
+const measlesDeath = { Total_Measles_Deaths: 'Measles_death' };
```

Comment:
Looks good to me.
Caution: I actually only reviewed the data sources code changes and the brief review the migration. I didn't actually check in detail if the map overlays are updated correctly according to their new data_source codes. But I assume you've smoke tested them 👍

Reply:
Thanks Chris. Yes I did check, will also let testers to do a sample test as well.